src/Ratchet/App.php

```diff
@@ -11,6 +11,7 @@
 use Ratchet\Server\FlashPolicy;
 use Ratchet\Http\HttpServer;
 use Ratchet\Http\Router;
+use Ratchet\WebSocket\MessageComponentInterface as DataComponentInterface;
```

Comment:
I think `as WsComponentInterface` would be a better name here. There is another interface in the RFC lib named `DataInterfae` which I wouldn't want to be conflated with here.